src/services/atomic-swap.service.js

```diff
@@ -30,6 +31,29 @@ const assembleTransaction = (partialTx, signatures, network) => {
   return proposal.prepareTx();
 };
 
+/**
+ * Creates the proposal on the Atomic Swap Service, handling errors that may occur on the process
+ * @param {string} partialTx Serialized PartialTx
+ * @param {string} password Password, length 3 or more
+ * @returns {Promise<{ proposalId: string }>} Returns the proposal identifier created by the service
+ */
+const serviceCreate = async (partialTx, password) => {
+  if (!partialTx) {
+    throw new Error('Invalid PartialTx');
+  }
+  if (!password || !password.length || password.length < 3) {
+    throw new Error('Password must have at least 3 characters');
+  }
+
+  const { success, id } = await swapService.create();
```

Comment:
Don't you need to use `partialTx` and `password` to create the swap?

Reply:
Fixed this on [d253b46](https://github.com/HathorNetwork/hathor-wallet-headless/pull/275/commits/d253b46751f6e6c64b6580ed98cbe304e6aa1995), along with the necessary url configuration on app init.

I'll add a comment about this as well.